### PR TITLE
Fix white flash when starting GameWindow based application on windows

### PR DESCRIPTION
**Pull request by peppy**
When loading, GameWindow (in windows) would be painted white until the first GL frame was painted to the window, which could be quite disconcerting. This change paints the window black at first opportunity. Not sure if we need/want further customisation of this colour, but black seems like an objectively better default than white.

### CODE REVIEW

File: src/OpenTK/Platform/Windows/WinGLNative.cs [hunk 1]
```diff
@@ -735,7 +735,7 @@ private IntPtr WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wPar
                     break;
 
                 case WindowMessage.ERASEBKGND:
-                    return new IntPtr(1);
+                    break;
```
Comment (Frassle) [inline src/OpenTK/Platform/Windows/WinGLNative.cs]: Once OpenGL is running this is a waste getting Windows to clear the background for us. Can this be turned off once the main loop is running.

Comment (peppy) [inline src/OpenTK/Platform/Windows/WinGLNative.cs]: I'll double-check on this, but I recall it only reaching this case once during execution.

Comment (Frassle) [inline src/OpenTK/Platform/Windows/WinGLNative.cs]: Ok if that's true can you add a comment here (in the code) explaining that behaviour.

Comment (peppy) [inline src/OpenTK/Platform/Windows/WinGLNative.cs]: yep, this should definitely be commented.

**Comment by varon** [comment]
>but black seems like an objectively better default than white.
Objectively I think you're right.

Thanks for the contribution, @peppy. We really appreciate the contributions you guys are adding.

**Comment by VPeruS** [comment]
What about other platforms? They have black or white?

**Comment by varon** [comment]
Given that the other platforms have APIs not designed by microsoft, I doubt they even have this behaviour at all. Good question, though.

**Comment by peppy** [comment]
I can confirm that MacOS flashes *red* as of recent. Probably also worth looking into.

**Comment by Nihlus** [comment]
Linux is, quite reasonably, black by default.

File: src/OpenTK/Platform/Windows/WinGLNative.cs (hunk 1 above)
Comment (varon) [inline src/OpenTK/Platform/Windows/WinGLNative.cs]: Soon as that's commented I'm happy with merging.

**Comment by Nihlus** [comment]
@peppy Any updates on the requested changes and Mac fix?

**Comment by varon** [comment]
@nihlus - That seems like a separate thing to address to this PR.

I'll give this a quick whirl on my mac along with the other stuff, then do the merge.

**Comment by peppy** [comment]
I still want to double-check there are no performance implications and add the comment regarding how this is triggered during execution. Should be able to confirm it tomorrow (my windows VM isn't working on my local PC at the moment)

**Comment by varon** [comment]
@peppy Sure. let me know when I can test this then.

**Comment by peppy** [comment]
While i can 100% confirm that Windows 10 only calls `ERASEBKGND` once, I can't say that about every windows version.

So i'm tempted to make it start returning `IntPtr.Zero` after frames begin rendering.. but not sure the best way to know when that state is reached (from `NativeWindowBase`). Are there any existing properties which would suit this purpose that I'm missing, or do you think it would be okay to add a `bool` that would become `true` after `ProcessEvents` is run once?

**Comment by Frassle** [comment]
Tested this on Windows 7. ERASEBKGND is called on startup and whenever we resize the window. Given it's not called on each refresh cycle I think that's acceptable. Would like a comment to the effect of "ERASEBKGND is called when the client area changes. This does not effect normal steady state performance." in the ERASEBKGND case clause.

**Comment by Frassle** [comment]
LGTM

**Pull request merged**